kratos/includes/element.h

```diff
@@ -981,6 +981,8 @@ class Element : public GeometricalObject, public Flags
                                             Matrix& rOutput,
                                             const ProcessInfo& rCurrentProcessInfo)
     {
+         if (rOutput.size1() != 0)
```

Comment:
Don' modify the base classes without the approval of the @KratosMultiphysics/technical-committee

Reply:
This follows the logic of other functions e.g. ```CalculateLocalSystem```: In case the function is not overwritten, one has to make sure, that the result is a matrix of size 0x0.

Reply:
@armingeiser we can take a look at this. However any change to the base classes should go in as a separate PR